zerver/lib/rate_limiter.py

```diff
@@ -67,15 +90,17 @@ def remove_ratelimit_rule(range_seconds, num_requests):
 def block_user(user, seconds, domain='all'):
     # type: (UserProfile, int, Text) -> None
     "Manually blocks a user id for the desired number of seconds"
-    _, _, blocking_key = redis_key(user, domain)
+    entity = RateLimitedUser(user, domain=domain)
+    _, _, blocking_key = list(entity.keys())
```

Comment:
Huh -- what was the lint error?

(Or maybe you're not seeing it anymore, since it looks like the new version of your branch doesn't add these `list` calls. In that case, it doesn't matter.)